.github/workflows/publish_inkSDK_ado.yml

```diff
@@ -22,10 +22,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Set up JDK 1.8
+      - name: Set up JDK 11
```

Comment:
The ink SDK doesn't need java 11 since it doesn't use WM right? Should we keep Java 1.8 for it? Maybe also change back its compileSdkVersion & targetSdkVersion so it works with java 8?

Reply:
Oh is not related to that. The problem is about that the Android Gradle Plugin requires Java 11.

https://github.com/microsoft/surface-duo-sdk/runs/4225041875?check_suite_focus=true
![image](https://user-images.githubusercontent.com/1050785/142035503-ee70552a-fb9d-47f9-8fe9-b410da5f36a7.png)

Reply:
Got it, thanks for the explanation!

Reply:
ofc! no problem Bianca! :-) thanks for the review! (it made me re-thing this too!)